src/utils/clack-utils.ts

```diff
@@ -185,17 +192,11 @@ export async function installPackage({
   sdkInstallSpinner.start(
     `${alreadyInstalled ? 'Updating' : 'Installing'} ${chalk.bold.cyan(
       packageName,
-    )} with ${chalk.bold(packageManager)}.`,
+    )} with ${chalk.bold(packageManager.getName())}.`,
   );
 
   try {
-    if (packageManager === 'yarn') {
```

Comment:
the `PackageManager` doesn't install `@latest` - I am not sure if we should do this but if there was a good reason for it, we might add it as default.